gbpservice/neutron/tests/unit/nsf/configurator/lib/filter_base.py

```diff
@@ -0,0 +1,384 @@
+import unittest
+
+
+class BaseTestCase(unittest.TestCase):
+    def __init__(self, *args, **kwargs):
+        super(BaseTestCase, self).__init__(*args, **kwargs)
```

Comment:
Need to change the path of unit tests to "gbpservice/nfp/test/unit/configurator/lib" directory.